docs/Simorgh-Release-Info.md

```diff
@@ -3,52 +3,52 @@ This table can then be generated using the following command: `node scripts/simo
-| mundo | [home](http://localhost:7080/mundo)<br/>[articles](http://localhost:7080/mundo/articles/ce42wzqr2mko)<br/>[MAP](http://localhost:7080/mundo/media-23283126)<br/>[PGL](http://localhost:7080/mundo/deportes-36935058)<br/>[STY](http://localhost:7080/mundo/23263889) | [home](https://www.test.bbc.com/mundo)<br/>[articles](https://www.test.bbc.com/mundo/articles/ce42wzqr2mko)<br/>[MAP](https://www.test.bbc.com/mundo/media-23283126)<br/>[PGL](https://www.test.bbc.com/mundo/noticias-23147451) | [home](https://www.stage.bbc.com/mundo)<br/>[articles](https://www.stage.bbc.com/mundo/articles/ce42wzqr2mko)<br/>[MAP](https://www.stage.bbc.com/mundo/media-23283126)<br/>[PGL](https://www.stage.bbc.com/mundo/noticias-23147451) | [home](https://www.bbc.com/mundo)<br/>[articles](https://www.bbc.com/mundo/articles/cdwrpl7qwqqo)<br/>[MAP](https://www.bbc.com/mundo/media-52123665)<br/>[PGL](https://www.bbc.com/mundo/deportes-36935058) | __Home__: 2nd April 2020<br/>__Articles__: Sept 2019<br/>__MAP__: 7th April 2020<br/>__PGL__: 24th March 2020 |
+| gahuza | [home](http://localhost:7080/gahuza)<br/>[articles](http://localhost:7080/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](http://localhost:7080/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](http://localhost:7080/gahuza/amakuru-23257470)<br/>[PGL](http://localhost:7080/gahuza/amakuru-43894701)<br/>[STY](http://localhost:7080/gahuza/23307435)<br/>[mostRead](http://localhost:7080/gahuza/popular/read) | [home](https://www.test.bbc.com/gahuza)<br/>[articles](https://www.test.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.test.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.test.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.test.bbc.com/gahuza/23111981)<br/>[STY](https://www.test.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.test.bbc.com/gahuza/popular/read) | [home](https://www.stage.bbc.com/gahuza)<br/>[articles](https://www.stage.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.stage.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.stage.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.stage.bbc.com/gahuza/23111981)<br/>[STY](https://www.stage.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.stage.bbc.com/gahuza/popular/read) | [home](https://www.bbc.com/gahuza)<br/>[articles](https://www.bbc.com/gahuza/articles/cryd02nzn81o)<br/>[liveRadio](https://www.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.bbc.com/gahuza/amakuru-51710168)<br/>[PGL](https://www.bbc.com/gahuza/amakuru-43894701)<br/>[mostRead](https://www.bbc.com/gahuza/popular/read) | __Home__: 19th Feb 2020<br/>__Articles__: Sept 2019<br/>__Live Radio__: 10th Jan 2020<br/>__MAP__: 27th Feb 2020<br/>__PGL__: 12th March 2020<br/>__STY__: 28th May 2020<br/>__Most Read__: 28th May 2020 |
```

Comment:
```suggestion
| gahuza | [home](http://localhost:7080/gahuza)<br/>[articles](http://localhost:7080/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](http://localhost:7080/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](http://localhost:7080/gahuza/amakuru-23257470)<br/>[PGL](http://localhost:7080/gahuza/amakuru-43894701)<br/>[STY](http://localhost:7080/gahuza/23307435)<br/>[mostRead](http://localhost:7080/gahuza/popular/read) | [home](https://www.test.bbc.com/gahuza)<br/>[articles](https://www.test.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.test.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.test.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.test.bbc.com/gahuza/23111981)<br/>[STY](https://www.test.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.test.bbc.com/gahuza/popular/read) | [home](https://www.stage.bbc.com/gahuza)<br/>[articles](https://www.stage.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.stage.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.stage.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.stage.bbc.com/gahuza/23111981)<br/>[STY](https://www.stage.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.stage.bbc.com/gahuza/popular/read) | [home](https://www.bbc.com/gahuza)<br/>[articles](https://www.bbc.com/gahuza/articles/cryd02nzn81o)<br/>[liveRadio](https://www.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.bbc.com/gahuza/amakuru-51710168)<br/>[STY](https://www.bbc.com/gahuza/amakuru-52821373)<br/>[PGL](https://www.bbc.com/gahuza/amakuru-43894701)<br/>[mostRead](https://www.bbc.com/gahuza/popular/read) | __Home__: 19th Feb 2020<br/>__Articles__: Sept 2019<br/>__Live Radio__: 10th Jan 2020<br/>__MAP__: 27th Feb 2020<br/>__PGL__: 12th March 2020<br/>__STY__: 28th May 2020<br/>__Most Read__: 28th May 2020 |
```

Reply:
The LIVE link was removed.

Reply:
yep, I just checked the cypress settings and we are missing links for LIVE for gahuza STY and ukchina

Reply:
resolved in latest commit